src/Model/Table/QueuedJobsTable.php

```diff
@@ -906,7 +906,7 @@ public function key(): string {
 		if ($this->_key !== null) {
 			return $this->_key;
 		}
-		$this->_key = sha1(microtime());
+		$this->_key = sha1(microtime() . rand(100, 999));
```

Comment:
Shouldn't we use mt_ ?

Reply:
today I learned... 😄

Reply:
Even though I also just read that from PHP 7.1+ forwards `rand()` uses the same number generator as `mt_rand()` so it doesn't matter 😄